FirebaseVertexAI/Sources/GenerativeAIService.swift

```diff
@@ -198,10 +187,16 @@ struct GenerativeAIService {
       }
     }
 
-    if let auth, let authToken = try await auth.getToken(forcingRefresh: false) {
+    if let auth = firebaseInfo.auth, let authToken = try await auth.getToken(
+      forcingRefresh: false
+    ) {
       urlRequest.setValue("Firebase \(authToken)", forHTTPHeaderField: "Authorization")
     }
 
+    if let googleAppID = firebaseInfo.googleAppID {
+      urlRequest.setValue(googleAppID, forHTTPHeaderField: "google-app-id")
```

Comment:
Is there any convention we need to follow such as `X-Firebase-AppId` or `x-goog-app-id`?

Reply:
It seems inconsistently named between Firebase products, but the x looks to be consistent. How about `X-Firebase-AppId` for clarity?

- RTDB: X-Firebase-GMPID
- Storage: x-firebase-gmpid
- Crashlytics: X-Crashlytics-Google-App-Id

Reply:
Okay, yeah if it's already inconsistent (and GMP is clearly outdated), I'm happy with `X-Firebase-AppId`.